TheWarWithin/DeathKnightUnholy.lua

```diff
@@ -150,20 +131,17 @@ me:RegisterResource( Enum.PowerType.Runes, {
             return t[ "time_to_" .. t.current + 1 ]
 
         elseif k == "time_to_max" then
-            return t.current == 6 and 0 or max( 0, t.expiry[6] - state.query_time )
-
-
-        elseif k == "add" then
```

Comment:
What was `t.add` for?  I genuinely don't know.  Is it referenced anywhere?

Reply:
Looks like a deprecated version of `gain()` to me, zero references to it in state file in the context of resources.